src/web/components/Picture.tsx

```diff
@@ -98,30 +128,39 @@ export const Img = ({
     isMainMedia = false,
     isLazy = true,
 }: Props) => {
-    const srcSetForRole = getSrcSetsForRole(role, imageSources);
-    const src = getFallbackSrc(srcSetForRole);
-    const sources = buildSourcesString(srcSetForRole);
-    const sizes = buildSizesString(role, isMainMedia);
+    const hdpiSources = getSources(role, 'hdpi', imageSources);
+    const mdpiSources = getSources(role, 'mdpi', imageSources);
+    const fallbackSrc = getFallback(role, 'hdpi', imageSources);
+    const sizes = getSizes(role, isMainMedia);
 
     return (
-        <img
-            itemProp="contentUrl"
-            alt={alt}
-            src={src}
-            srcSet={sources}
-            sizes={sizes}
-            height={height}
-            width={width}
-            loading={isLazy && !Img.disableLazyLoading ? 'lazy' : undefined}
-            // https://stackoverflow.com/questions/10844205/html-5-strange-img-always-adds-3px-margin-at-bottom
-            // why did we add the css `vertical-align: middle;` to the img tag
-            className={css`
-                vertical-align: middle;
-            `}
-        />
+        <picture itemProp="contentUrl">
+            {/* HDPI Source (DPR2) - images in this srcset have `dpr=2&quality=45` in the url */}
+            <source
+                srcSet={hdpiSources}
+                sizes={sizes}
+                media="(-webkit-min-device-pixel-ratio: 1.25), (min-resolution: 120dpi)"
+            />
+            {/* MDPI Source (DPR1) - images in this srcset have `quality=85` in the url */}
+            <source srcSet={mdpiSources} sizes={sizes} />
+            <img
+                alt={alt}
+                src={fallbackSrc}
+                height={height}
+                width={width}
+                loading={
+                    isLazy && !Picture.disableLazyLoading ? 'lazy' : undefined
```

Comment:
Ah! We should start using the `loading` attribute in image-rendering too!

Reply:
+1 This was one of my 'what do the open source image components do' points! It's a big perf win I think